client/packages/common/src/hooks/useDialog/useDialog.tsx

```diff
@@ -161,6 +166,9 @@ export const useDialog = (dialogProps?: DialogProps): DialogState => {
       isRtl,
       animationTimeout
     );
+    const { isOpen: keyboardIsOpen } = useKeyboardContext();
+    const isSmallerScreen = useMediaQuery('(max-height: 850px)');
```

Comment:
Felt like a reasonable height to be okay with sticking with modals as they are, most tablets around 750-800 high... happy to play around with this